recipe/meta.yaml

```diff
@@ -38,14 +40,16 @@ requirements:
   host:
     - r-base
     - r-gmp >=0.5_8
-    - mpfr        # [not win]
-    - m2w64-mpfr  # [win]
+    - gmp                          # [not win]
+    - mpfr                         # [not win]
+    - {{ native }}gmp              # [win]
+    - {{ native }}mpfr             # [win]
   run:
     - r-base
     - {{ native }}gcc-libs         # [win]
     - r-gmp >=0.5_8
-    - mpfr        # [not win]
-    - m2w64-mpfr  # [win]
+    - {{ native }}gmp              # [win]
+    - {{ native }}mpfr             # [win]
```

Comment:
This is rather annoying - the pinnings for non-windows do not work with the `{{ native }}` variable. Fortunately, this is going away, but this recipe will need to once again be reworked when the migrator comes through.